### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by Guidetteliv11** [community/community#28133] [Mobile] [answered]
I wanna say thank you i really like this app i need help catching someone taking data off my phone please continue to help lol 💕💕 #23204 

`

**Comment by stevepopovich** [comment, marked as answer]
Hi @Guidetteliv11, thanks for the love. I don't think we at GitHub can help you with that. Sorry!

**Reply by Guidetteliv11** [reply]
Hey Haha thank you lol
